### PR TITLE
delete abstract and description from preface

### PR DESCRIPTION
**Pull request by kswanson33**

### CODE REVIEW

File: lib/kitchen/directions/bake_preface/v1.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ module Kitchen::Directions::BakePreface
   class V1
     def bake(book:, title_element:)
       book.pages('$.preface').each do |page|
+        page.search('div[data-type="description"], div[data-type="abstract"]').each(&:trash)
```
Comment (brittany-johnson) [inline lib/kitchen/directions/bake_preface/v1.rb:7]: Is this going to negatively affect other books currently using v1?

Comment (kswanson33) [reply lib/kitchen/directions/bake_preface/v1.rb:7]: good question... idk lol 😬

Comment (kswanson33) [reply lib/kitchen/directions/bake_preface/v1.rb:7]: so I double checked chem and uphysics, and neither of them will be impacted. i also created another test case/extended the case to verify that this doesn't make `BakePreface` blow up without these elements. but i don't know about other books.

Comment (brittany-johnson) [reply lib/kitchen/directions/bake_preface/v1.rb:7]: I just check Am Gov and doesn't look like it'll be affected either. 

But this does give me the idea that maybe we should be running recipe spec as part of github checks in kitchen repo?

**Review by brittany-johnson — APPROVED** [review]

**Pull request merged**